subsys/llext/CMakeLists.txt

```diff
@@ -1,5 +1,6 @@
 if(CONFIG_LLEXT)
   zephyr_library()
+  zephyr_library_compile_definitions(-D_POSIX_C_SOURCE=200809L)
```

Comment:
Can you add a comment here to clarify what is this used for and where?

Reply:
Sure, will do now